windows/tag_window.py

```diff
@@ -31,3 +39,12 @@ def use_scenario_names_for_script_names(self):
                         flags.use_scenario_names_in_scripts)
         except Exception:
             return False
+
+
+class HaloConfigWindow(ConfigWindow):
+
+    def post_toplevel_init(self):
+        if not is_path_empty(e_c.MOZZ_ICON_PATH):
+            self.iconbitmap_filepath = e_c.MOZZ_ICON_PATH

```

Comment:
Don't we need a try-except here to avoid this process exceptioning under Linux?

Reply:
why would that exception? is that property not defined on linux?

Reply:
Linux tkinter not supporting directly assigning icon paths as Window icons. It's the whole reason for all of those try except blocks or did I understand that wrong?